src/main/java/com/wire/xenon/models/otr/Missing.java

```diff
@@ -1,26 +1,35 @@
 package com.wire.xenon.models.otr;
 
+import com.wire.xenon.backend.models.Qualified;
+
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
 
-//<UserId, [ClientId]>
-public class Missing extends ConcurrentHashMap<UUID, Collection<String>> {
-    public Collection<String> toClients(UUID userId) {
+/**
+ * Structure to handle users and devices (clients) belonging to them.
+ *
+ * <p>
+ *     Usually holds the users and their devices when a message needs to be sent.
+ *     The sender will need to fetch all the devices that need to get notified inside a conversation,
```

Comment:
Yep, that is misleading, removed the "usually" and tried to explain it better.
In general, I really wanted to move some files to different packages and/or rename them but everything is part of the API shared with other projects. I changed the API enough and did not want to cause more trouble in a single version update. If we have the will and capacity we might rework how this files are structured but it will need to be propagated to many repos (helium, legalhold, lithium, roman)